api/v1alpha2/sharedazuredbs_types.go

```diff
@@ -77,7 +77,7 @@ type PSQLStorageProfile struct {
 	GeoRedundantBackup psql.GeoRedundantBackup `json:"geoRedundantBackup,omitempty"`
 	// StorageMB - Max storage allowed for a server.
 	StorageMB *int32 `json:"storageMB,omitempty"`
-	// StorageAutogrow - Enable Storage Auto Grow. Possible values include: 'StorageAutogrowEnabled', 'StorageAutogrowDisabled'
+	// StorageAutogrow - Enable Storage Auto Grow. Possible values include: 'Enabled', 'Disabled'
 	StorageAutogrow psql.StorageAutogrow `json:"storageAutogrow,omitempty"`
```

Comment:
If you also annotate this with:
`// +kubebuilder:validation:Enum=Enabled,Disabled`, it'll reject other values at YAM: submit time